Exercise_B/index.html

```diff
@@ -9,8 +9,9 @@
     <link rel="stylesheet" href="index.css" />
   </head>
   <body>
-    <!-- CODE HERE -->
-
+    <div class="blue-box"></div>
+    <div class="green-box"></div>
+    <div class="red-box"></div>
     <script src="" async defer></script>
   </body>
 </html>
```

Comment:
Hello Asis
Great use of using Flex here, they are positioned in the right place using Flex.